src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -6138,3 +6138,63 @@ def test_aks_create_and_update_ssh_public_key(self, resource_group, resource_gro
         # delete
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='eastus2euap')
+    def test_node_public_ip_tags(self, resource_group, resource_group_location):
```

Comment:
I [tested](https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=63836715&view=results) the case but it failed on feature not registered. Could you please add custom headers to the case?

Reply:
done